platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/state/MerkleStateRootTest.java

```diff
@@ -1067,4 +1069,68 @@ void migrate_platform_absent() {
             assertThrows(IllegalStateException.class, () -> stateRoot.migrate(CURRENT_VERSION - 1));
         }
     }
+
+    @Nested
+    @DisplayName("Hashing test")
+    class HashingTest {
+
+        @BeforeEach
+        void setUp() {
+            setupAnimalMerkleMap();
+            setupSingletonCountry();
+            setupSteamQueue();
+
+            add(fruitMerkleMap, fruitMetadata, A_KEY, APPLE);
```

Comment:
Do we need to call `setupFruitMerkleMap` before adding to it?

Reply:
Yes, I'll added it

Reply:
Fixed